testing/src/cypress/e2e/04-api-institution/02-instition-create-update-location.cy.ts

```diff
@@ -2,19 +2,41 @@ import InstitutionHelperActions from "../../custom-command/institution/common-he
 import Authorization, {
   ClientId,
 } from "../../custom-command/common/authorization";
+import axios from "axios";
 
 const institutionHelperActions = new InstitutionHelperActions();
 
 const USERNAME = institutionHelperActions.getUserNameForApiTest();
 const PASSWORD = institutionHelperActions.getUserPasswordForApiTest();
 const TOKEN_URL = institutionHelperActions.getApiUrlForKeyCloakToken();
 
-function createOrUpdateInstitutionLocation(
+async function getUniqueInstitutionCode(token: string) {
+  let locationIds: string[] | PromiseLike<string[]> = [];
```

Comment:
I would expect that the `locationIds` would be an array of numbers. Can you please elaborate on why it is also a `PromiseLike<string[]>`?

Reply:
Fixed it just needs to be a array of strings, i was thinking that the locationIds needs to be  fetched to check it against the newly generated Id.